ansible_runner/utils/capacity.py

```diff
@@ -0,0 +1,17 @@
+import multiprocessing
+import resource
+
+
+def get_cpu_capacity():
+    # `multiprocessing` info: https://docs.python.org/3/library/multiprocessing.html
+    forkcpu = 4
```

Comment:
should we return raw cpu count here, and then on the control plane side calculate the capacity based on the user settings for `SYSTEM_TASK_FORKS_CPU`?

https://github.com/ansible/awx/blob/d89719c7409528006a65378c3e6ae1a4b2666ec0/awx/main/utils/common.py#L719